ccswbs/gus
# Remove blue background from Image Overlay

# Summary of changes
Remove code for blue background and associated image.

## Frontend
Remove code for blue background and associated image.

## Backend
- Removed the term for the Blue background from the Image Overlay Styles vocabulary.

# Test Plan
- Run the PR branch against the live site
- Ensure nothing breaks
- Ensure blue background is no longer an option for the Image overlay
